config/application.properties

```diff
@@ -118,4 +118,5 @@ pn.interop.base-url=https://auth.uat.interop.pagopa.it
 pn.interop.token-oauth2.path=/token.oauth2
 
 
-
+#TIMING MANAGEMENT FOR TIMELINE EVENTS
+pn.config.timing.timeline.tuning-value=2
```

Comment:
darei a questa properties un nome più riconoscibile del tipo 
pn.configuration.waiting.timing.slow.multiplier=2 
e và inserita nei file timing-"EVN".properties (es. timing-dev.properties) 
così da poterla variare per ambiente.  (per il momento va bene in tutti i file settata a 2)

Reply:
modifica apportata come rischiesto